drivers/can/Kconfig.xlnx

```diff
@@ -0,0 +1,17 @@
+#
+# Xilinx Processor System CAN controller driver
+# configuration options
+#
+# Copyright (c) 2022, Weidmueller Interface GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_XLNX_CAN := xlnx,zynq-can-1.0
+
+config CAN_XLNX
+	bool "Xilinx Processor System CAN controller driver"
+	default $(dt_compat_enabled,$(DT_COMPAT_XLNX_CAN))
+	depends on SOC_XILINX_ZYNQMP_RPU || SOC_FAMILY_XILINX_ZYNQ7000
+	depends on !QEMU_TARGET
```

Comment:
remove `DT_COMPAT_XLNX_CAN`

and make this:

```
default y
depends on DT_HAS_XLNX_ZYNQ_CAN_1_0_ENABLED
```

remove `depends on SOC_XILINX...`